### ISSUE TITLE
setStatus to ready doesn't work when setStatusAutomatically is true

### ISSUE THREAD

**Issue by wesleyhf** [Adyen/adyen-web#1331]
**Describe the bug**
When I try to setStatus to `ready` inside of `onSubmit` it is not work when `setStatusAutomatically` is `true`.

**To Reproduce**
1. set `setStatusAutomatically` to `true`
2. inside of `onSubmit` try to call `dropin.setStatus('ready');`

**Expected behavior**
After the `onSubmit` call to my back-end I want to set the `dropin` to `ready` state.

**Desktop (please complete the following information):**
 - OS: MacOS 11.6
 - Browser: Chrome
 - Version: 94.0.4606.71 (Official Build) (x86_64)

**Additional context**

```js
const checkout = new AdyenCheckout({
    setStatusAutomatically: true,
    onSubmit: (state, dropin) => {
        handleRequest(state, dropin);
        dropin.setStatus('ready');
    },
});
```

**Comment by wesleyhf** [comment]
Hello @ribeiroguilherme, sorry for it 😅 .

I was trying to use the `4.9.0` version, through the CDN files:

- https://checkoutshopper-test.adyen.com/checkoutshopper/sdk/4.9.0/adyen.js
- https://checkoutshopper-test.adyen.com/checkoutshopper/sdk/4.9.0/adyen.css

**Comment by ribeiroguilherme** [comment]
Hi @wesleyhf ,

Indeed there is an issue with `setStatusAutomatically` with our Drop-in component. I created a ticket in our backlog to fix it. Thanks for letting us know. I will update this ticket as soon as we fix it.

If you would like to still use it with `setStatusAutomatically: true`, here is an workaround:

```javascript
const checkout = new AdyenCheckout({
    setStatusAutomatically: true,
    onSubmit: async (state, dropin) => {
        dropin.setStatus('loading');
        await handleRequest(state, dropin);
        dropin.setStatus('ready');
    },
});
```

**Comment by ribeiroguilherme** [comment]
Hi @wesleyhf ,

We released the version 5.6.2 which contains the bugfix for this issue. Thanks again for reaching out!
